crates/ruff/src/rules/flake8_pyi/rules/pass_statement_stub_body.rs

```diff
@@ -1,18 +1,23 @@
 use rustpython_parser::ast::{Ranged, Stmt};
 
-use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_diagnostics::{AlwaysAutofixableViolation, Diagnostic, Edit, Fix};
 use ruff_macros::{derive_message_formats, violation};
 
 use crate::checkers::ast::Checker;
+use crate::registry::Rule;
 
 #[violation]
 pub struct PassStatementStubBody;
 
-impl Violation for PassStatementStubBody {
+impl AlwaysAutofixableViolation for PassStatementStubBody {
     #[derive_message_formats]
     fn message(&self) -> String {
         format!("Empty body should contain `...`, not `pass`")
     }
+
+    fn autofix_title(&self) -> String {
+        format!("Replace `pass` with `...` in empty function body")
```

Comment:
Could be empty class or function body, right?

Reply:
I think "Replace with `...`" is sufficient here